tests/install/conftest.py

```diff
@@ -98,7 +98,15 @@ def installer_iso(request):
 @pytest.fixture(scope='function')
 def system_disks_names(request):
     firmware = request.getfixturevalue("firmware")
-    yield {"uefi": "nvme0n1", "bios": "sda"}[firmware]
+    system_disk_config = request.getfixturevalue("system_disk_config")
```

Comment:
`system_disk_config` could be validated here.

Maybe use an enum to represent the possible values?

Also `disk` could have a more explicit name like `single_disk`.

Reply:
> `system_disk_config` could be validated here

do you mean type-hinted?

> Also `disk` could have a more explicit name like `single_disk`.

I am not very happy with `disk` but those land in the test name, so I'd prefer a short alternative.  For context, after piling IPv6 parameter on top of this, the test are like:
```
tests/install/test.py::TestNested::test_boot_inst[uefi-83nightly-host1-disk-iso-nosr-ipv6static]
```

And we are not anywhere near what they ought to be :frowning_face:

Reply:
> > `system_disk_config` could be validated here
> 
> do you mean type-hinted?

I mean validated at run time to make sure that the value is a valid one.

An enum could help also with the static analysis. I think that mypy/pyright are able to check when we are not checking all the possible variants